packages/dd-trace/test/util.spec.js

```diff
@@ -1,5 +1,6 @@
 'use strict'
 
+const { expect } = require('chai')
```

Comment:
This file wasn't otherwise changed so let's revert this.

Reply:
oops 😅 auto-import